### DISCUSSION TITLE
Stuck while "punching holes"

### DISCUSSION THREAD

**Discussion by tyson-dev** [isid0re/SoloLeveling#322] [General]
Seen this 3 times now with different chars, i think everytime with crystal swords.

He goes up to larzuk, the slot where you have to put the item opens and then he walks over to stash trying to move the item from stash to the still visible Slot/Socket thing and then gets stuck with the crystal sword in hand/cursor.

Looks like he doesn't move the wanted item to inventory beforehand and then tries to move it from stash to the "socket menu".

**Comment by mentoss11** [comment]
I have the same problem too,cs and larzuk sox  q crash game

**Comment by isid0re** [comment]
thank you I am checking this out and will post a fix soon.

**Reply by isid0re** [reply]
This is fixed as of the current push

**Reply by tyson-dev** [reply]
This is still a thing, smells like he's messing up Game/Position with Packet-Casting maybe.

This time Druid Wind/Broad Sword. Again first walks to larzuk, opens up the Socket/Anvil Menu, just then walks over to stash and picks the Broadsword ( with the socket-menu visible all the time) and then stuck forever.

When i pause the bot and try to do something i get rubberbanded back to stash position. ( e.g.: walk up to malah she's not there, cannot click on stash/doesn't open).

Also i noticed he still had an unidentified item in inventory so maybe selling and clearjunk before socketing may help?

Also also maybe This is only an issue in the first run normal. When i save/exit he does seemingly do the socket quest correctly next time, as he has a 4sox broad sword now but he doesn't console log i guess :D

**Reply by isid0re** [reply]
Thanks your description really helped isolate the problem. I wasn't specifically calling openStash() so it would bug out as you described. I've corrected it and tested it about 10 times (5 crystal swords (spirit) and 5 voulge (insight). As it appears to work as intended now I've  pushed it to the repo. 

Thanks again and please let me know if it works for you.

**Reply by tyson-dev** [reply]
Thanks for your fast response and fix M8, glad i could help a little, will keep watching.

**Reply by tyson-dev** [reply]
Hey M8, can confirm it works now, Cold-Sorc just smoothly punched a Broad Sword first try @ level 33, thumbs up and thanks for the quick fix again.
